src/ngraph/frontend/onnx_import/onnx.cpp

```diff
@@ -52,7 +52,32 @@ namespace ngraph
             } // namespace error
         }     // namespace detail
 
-        std::shared_ptr<Function> import_onnx_model(std::istream& sin, const Weights& weights)
+        static void update_external_data_paths(onnx::ModelProto& model_proto,
```

Comment:
It is worth to consider if we want change onnx API to OPP version. In such case passing `model_path` as additional argument can be replaced by class field introduction.